core/src/main/java/org/apache/xtable/delta/DeltaConversionTarget.java

```diff
@@ -268,9 +281,23 @@ private void commitTransaction() {
     }
 
     private Map<String, String> getConfigurationsForDeltaSync() {
+
```

Comment:
We can test this [here](https://github.com/apache/incubator-xtable/blob/main/core/src/test/java/org/apache/xtable/delta/ITDeltaConversionTargetSource.java)

I would expect we can add the timestamp_ntz and see that the delta table has the correct versions set for min reader/writer?